### DISCUSSION TITLE
s+block code = block

### DISCUSSION THREAD

**Discussion by Sreehari521** [ScratchAddons/ScratchAddons#3107] [Ideas] [answered]
### What are that block codes?
* fvr means forever
* if_els means if then else
* if means if then
* x means change x by
* y means change y by 
* etc,etc,etc

**Comment by mxmou** [comment, marked as answer]
This already exist - enable developer tools and press Ctrl+Space to search for blocks.

**Reply by Sreehari521** [reply]
K

**Comment by Genius398** [comment]
> ## If we press S + Any Block Code It should give us that block
> ### What are that block codes?
> * fvr means forever
> * if_els means if then else
> * if means if then
> * x means change x by
> * y means change y by
> * etc,etc,etc

You can also press the scroll wheel
